index.html

```diff
@@ -0,0 +1,274 @@
+          <div class="title">
+            <h2>Faithless live on Exit Festival 2015 [ Minimal Techno Mix ]</h2>
+          </div>
+          <div class="options-video">
+            <div class="views">
+              <span class="video-text">150,484 views</span>
+
+              <div class="right-video-icons">
+                <ul class="video-icons">
+                  <li>
+                    <img class="icons" alt="icon" src="images/like.png" /><span
+                      class="span-text"
+                      >1135</span
+                    >
+                  </li>
+                  <li>
+                    <img class="icons" alt="icon" src="images/dislike.png" />
+                    <span class="span-text"> 322</span>
+                  </li>
+                  <li>
+                    <img class="icons" alt="icon" src="images/share.png" />
+                  </li>
+                  <li>
+                    <img class="icons" alt="icon" src="images/save.png" />
+                  </li>
+                  <li>
+                    <img class="icons" alt="icon" src="images/more.png" />
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </div>
+          <hr />
+        </div>
+        <div class="sidebar">
+          <div class="main-recommend-section">
+            <p class="up-next">Up Next</p>
+            <span class="text-autoplay">Autoplay</span
+            ><img src="images/toggle-on.png" alt="autoplay" class="toogle" />
+          </div>
+        </div>
+
+        <div class="recommend">
+          <div class="recomended-image">
+            <img src="images/thumbnail-1.jpg" alt="video1" />
+          </div>
+          <div class="sub-text-video">
+            <p class="text-video">
+              Why he would to it? <br />Nimes is active for a sound.
+            </p>
+
+            <p class="sub-text-user">Mark Fritz</p>
+            <p class="views-side">1K views</p>
+          </div>
+        </div>
+        <div class="recommend">
+          <div class="image--2">
+            <img src="images/thumbnail-1.jpg" alt="video1" />
+          </div>
+          <div class="sub-text-video text--1">
+            <p class="text-video">
+              Why he would to it? <br />Nimes is active for a sound.
+            </p>
+
+            <p class="sub-text-user">Mark Fritz</p>
+            <p class="views-side">1K views</p>
+          </div>
+        </div>
+        <div class="recommend">
+          <div class="image--3">
+            <img src="images/thumbnail-1.jpg" alt="video1" />
+          </div>
+          <div class="sub-text-video text--2">
+            <p class="text-video">
+              Why he would to it? <br />Nimes is active for a sound.
+            </p>
+
+            <p class="sub-text-user">Mark Fritz</p>
+            <p class="views-side">1K views</p>
+          </div>
+        </div>
+      </main>
+
+      <div class="profile-section">
```

Comment:
It should be a section tag instead of div.
When you are starting a new section it is important to use a section tag.